Description: Fixes the compatibility with the version of ASM in Debian
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/src/java/edu/umd/cs/findbugs/classfile/engine/asm/FindBugsASM.java
+++ b/src/java/edu/umd/cs/findbugs/classfile/engine/asm/FindBugsASM.java
@@ -26,6 +26,6 @@
  */
 public class FindBugsASM {
 
-    public static final int ASM_VERSION = Opcodes.ASM6;
+    public static final int ASM_VERSION = Opcodes.ASM5;
 
 }
